src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -3587,121 +3589,97 @@ internal void VerifyValueType(object? value)
             }
 
             Debug.Fail("Error result not expected");
-            return false;
+            return default;
```

Comment:
This seems like an odd choice. Is there value here in converting this from `false` to `default`?

Reply:
Not really, just a convention since the return value doesn't matter. Making it `false` instead makes it more readable and avoids this question so I'll change it.